clang/lib/AST/ASTImporter.cpp

```diff
@@ -1551,7 +1551,7 @@ ASTNodeImporter::VisitCountAttributedType(const CountAttributedType *T) {
   Expr *CountExpr = importChecked(Err, T->getCountExpr());
 
   SmallVector<TypeCoupledDeclRefInfo, 1> CoupledDecls;
-  for (auto TI : T->dependent_decls()) {
+  for (const TypeCoupledDeclRefInfo &TI : T->dependent_decls()) {
```

Comment:
```suggestion
  for (const auto &TI : T->dependent_decls()) {
```

Reply:
Thank you @JOE1994 for reviews. I have updated patch.

Reply:
Was there a reason to suggest it back to `auto` from `TypeCoupledDeclRefInfo`? I thought Clang prefers to have an explicit type name when the deduced type is not explicit from RHS.

Reply:
Done. Thanks for the review's !

Reply:
I agree `TypeCoupledDeclRefInfo` should be kept. The `auto` here is not clear.